web/src/app/components/bar-chart/bar-chart.component.ts

```diff
@@ -344,7 +371,7 @@ export class BarChartComponent implements AfterViewInit {
       this.yAxis.select('.domain').remove();
 
       newBars
-        .attr('class', '.newBars')
+        .attr('class', 'new-bars')
         .enter()
         .append('rect')
           .attr('x', (d: Item) => this.xScaleFocus(d.startTime))
```

Comment:
You're adding extra indentation after append for some places while using the same indentation for other places, could you make them consistent?

Reply:
Please see the comment below!